helm_deploy/values-prod.yaml

```diff
@@ -3,7 +3,7 @@
 
 generic-service:
   ingress:
-    host: support-additional-needs-api.hmpps.service.justice.gov.uk
+    host: support-for-additional-needs-ap.hmpps.service.justice.gov.uk
```

Comment:
should be? support-for-additional-needs-api.hmpps.service.justice.gov.uk

Reply:
Good spot .... me and my copy&paste! 🤣